README.md

```diff
@@ -2,14 +2,12 @@
 Ansible automation to load test-data of various sample applications to aid OCP Migration development efforts
 
 # Usage
+For backups/restores ark is required, it can be install using playbooks from this repo ```https://github.com/fusor/ocp-velero-ansible```
 
-Copy config.yml.example to config.yml and set variables
-
-Export KUBECONFIG
+Export KUBECONFIG for example: ```export KUBECONFIG=~/.kube/config```
 
 Set cluster password to env variable like this:
 ```export kubeadmin_password="password"```
 
-Run ```ansible-playbook cake_php.yml``` to deploy a sample app
-
-To clean after app deployment you can do ```oc delete all --selector app=cakephp-example```
\ No newline at end of file
+Use ```ansible-playbook login-ocp.yml``` to login in cluster
```

Comment:
Alternatively you can just put this role in the `cake-php.yml` playbook so that it runs everytime you want to deploy the app. This will ensure the user is always logged into the right cluster.